### ISSUE TITLE
How to setup Kinect Version 1 with Processing 3.5.4 on OSX 10.9.5

### ISSUE THREAD

**Issue by vowelmusic** [totovr/SimpleOpenNI#29]
### Posting rules

1. **Fill** the **Your System Configuration section (all of it!)** if you have some kind of error or performance question.
2. **No duplicated** posts.
3. **No** posts about **questions already answered / clearly explained in** the **documentation**
4. Set a **proper issue title**: add the OSX/Ubuntu/Windows word and be specific (e.g. do not simple call it: `Compile error`).
5. Only English comments.
Issues/comments which do not follow these rules will be **ignored or removed** with no further clarification.

### Type of Issue
You might select multiple topics, delete the rest:
- Compilation/installation error
- Execution error
- Help wanted
- Question
- Enhancement / offering possible extensions / pull request / etc
- Other (type your own type)

### Your System Configuration
**Operating system**:
**Version of Kinect**:

**Comment by vowelmusic** [comment]
Im trying to get the basic skeleton tracking. New to this but have openKinect working fine. Are there other libraries I need to install in order for SimpleOpenNI to work maybe?
Thanks.

**Comment by vowelmusic** [comment]
FWIW I got it working using Processing 2.2.1 & the appropriate SimpleOpenNi lib.
ATB.

**Comment by chuanisawesome** [comment]
Hi @vowelmusic how did you get SimpleOpenNI to work? I have processing 3.5 installed and I always get an error message about the library not being installed properly. The way I installed it is by downloading the github and unzip and copying it over to the library folder. Please help I would like to have the hand tracking working for a project. Thank you!

**Comment by vowelmusic** [comment]
I'm sorry I can't remember much about it now...
FWIW I got all my info from this youtube channel:
https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZMlWHdcy8hAGDy6IaoxUKf
Hope this helps,
L.

**Comment by chuanisawesome** [comment]
Hi @vowelmusic thank you! I watched the videos before and it doesnt use the SimpleOpenNI. Im just getting an error message that says libSimpleOpenNI.jnilib is not trusted and even though I trust it through the settings it won't work. :/

But thank you for the response

**Comment by vowelmusic** [comment]
I could only get the skeleton tracking stuff to work with Processing 2.2.1 on my machine.

**Comment by chuanisawesome** [comment]
@vowelmusic I will try with the 2.2.1 👍 and I will let you know :) thank you!

**Comment by chuanisawesome** [comment]
Yeah I still get "“libSimpleOpenNI.jnilib” cannot be opened because the developer cannot be verified." and Ive also accepted through my apple settings and the library still doesn't work.

**Comment by vowelmusic** [comment]
Maybe this will help?:
https://support.apple.com/en-gb/guide/mac-help/mh40616/mac

**Comment by vowelmusic** [comment]
Or try 'Get Info' on the file and change some settings there.
Other than that I don't know...

**Comment by vowelmusic** [comment]
Sorry to be so vague, but I think I may have had a similar issue - sure it was in the 'Get Info' settings...
Good luck.

**Comment by vowelmusic** [comment]
Make sure the locked box isn't checked & change the permissions at the bottom...

**Comment by chuanisawesome** [comment]
@vowelmusic THANK YOU! I keep forgetting to unlock for permission and then locking it again. It is working fine now, I will test more with the Kinect but as of now it doesnt throw an error.

**Comment by vowelmusic** [comment]
Happy days..:)
